roles/Stackstorm.post_install/tasks/packs.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: get authentication token
+  command: st2 auth "{{ st2_auth_username }}" -p "{{ st2_auth_password }}" -t
+  register: st2_token
```

Comment:
Having st2 pack install functionality in `st2` role comes with a bonus that we don't need to generate a token as additional step.

Reply:
yea makes sense

Reply:
I actually don't see where this is generated before I need it.

Reply:
See https://github.com/StackStorm/ansible-st2/blob/684d0c3052b8fc3503baec06c9eaed3fd32e9d6b/roles/StackStorm.st2/tasks/auth.yml#L49-L68

`$HOME/.st2/config` file contains username/password pair meaning no further auth is required from st2client side.

Reply:
I see it. I think I didn’t setup as root. How do you feel about ignore EL6/ Centos6 - and just not running it ? The failure for that is due to the map function to see if packs are installed not standard in EL6 @armab

Reply:
We'll deprecate EL6 in next 1-2 st2 releases so I'm good with that.